source/reference/data-types.txt

```diff
@@ -60,17 +60,34 @@ Starting in  1.8.0, the ``ObjectId`` wrapper no longer accepts:
 
    :method:`ObjectId` 
 
+Double 
+------
+
+The :node-api:`Double() <Double.html>` constructor can be used to
+explicitly specify a double:
+
+.. code-block:: javascript
+
+      db.types.insertOne(
+         {
+            "_id": 2,
+            "value": Double("1"),
+            "expectedType": "Double" 
+         }
+      )
+
+.. note:: 
+
+   If a number can be converted to a 32-bit integer, ``mongosh`` will store it 
+   as ``Int32``. If not, ``mongosh`` defaults to storing the number as a 
+   ``Double``. Numerical values that are stored as ``Int32`` in ``mongosh`` 
+   would have been stored by default as ``Double`` in the ``mongo`` shell.
```

Comment:
This paragraph now sounds like it also applies to values like `Double(1)`, not just `1`, but using `Double()` and `Int32()` explicitly is intended as the way out of that.

Can we make it clearer that this is only talking about JS number values, and maybe add something like "Use the Double() or Int32() constructors to unambiguously specify the value type." at the end to indicate to users that these are the intended escape hatches?

Reply:
Will do. Thanks for the suggestion!